pkgs/development/python-modules/meraki/default.nix

```
@@ -8,14 +8,14 @@
 
 buildPythonPackage rec {
   pname = "meraki";
-  version = "1.39.0";
+  version = "1.41.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/meraki/dashboard-api-python/commit/559e4d3edf9be21c4ee2bb4091afe55ba5aa5b5b